charts/pentaho-carte/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
it seems like pentaho-carte is not really built for cloud-native... 😢 Without knowing pentaho-carte in detail I also have no better ideas for a test.

if you don't get this test working, you can delete the test file (and folder) and skip the chart testing as a last resort.

Reply:
decided to keep the test file executing 'curl --help' as either connection fails (wget --header ... ) or command not allowed (curl --user ...), see also comments in test-connection.yaml lines 17ff.